### ISSUE TITLE
Fedora Dynamic module builds for Nginx 1.15.8

### ISSUE THREAD

**Issue by chienhsingwu** [slact/nchan#544]
Hi,

We would like to use the Fedora dynamic lib build to host nchan on RedHat OS 7, with nginx 1.15.8, the curretn version. Does the link in https://nchan.io/#install support the OS and nginx version? If not, is this something you can help us, or let me know how to produce the dynamic lib?

Thank you!

**Comment by chienhsingwu** [comment]
Hi, really appreciate your advice on the question!

**Comment by dvershinin** [comment]
@chienhsingwu you can find a handful of modules [here](https://extras.getpagespeed.com/redhat/7/x86_64/repoview/letter_n.group.html), including [nchan](https://extras.getpagespeed.com/redhat/7/x86_64/repoview/nginx-module-nchan.html). (the repository requires subscription, support CentoS/RHEL 6, 7, 8)

**Comment by chienhsingwu** [comment]
@dvershinin, thanks for tips. My understanding is that the dynamic modules needs to be built with the same configuration parameters (identified by nginx -v command) as the targeting nginx executable. Is that also your understanding? Does this nchan module target the nginx build in the same repo: https://extras.getpagespeed.com/redhat/7/x86_64/repoview/nginx.html?

**Comment by dvershinin** [comment]
@chienhsingwu that is correct. Technically, it should be also compatible with any same NGINX versions which were built with different configuration parameters, *as long* as `--with-compat` switch is provided.

Yes, the nchan module build targets NGINX build in the same repo. It is also compatible with official nginx.org's RPM builds, e.g.

* You can have just the GetPageSpeed repository, and install NGINX + any module available separately (nchan is one of them) to set up your desired NGINX stack. Sort of constructor. You don't have to install every module, just the ones you need
* If you already have NGINX installed from the official nginx.org RPM repository, you don't have to remove your NGINX. Simply add GetPageSpeed repo and install the modules you need  - this will just work as well, because nginx.org's NGINX is built with the same set of configuration parameters as GetPageSpeed one. In other words, all the modules in GetPageSpeed repo are compatible to either nginx.org's NGINX and GetPageSpeed's NGINX.

**Comment by chienhsingwu** [comment]
Thanks @dvershinin !
